RevenueCatUI/CustomerCenter/Views/CustomerCenterNavigationOptions.swift

```diff
@@ -34,26 +34,52 @@ public struct CustomerCenterNavigationOptions {
     /// - `false`: Does not display a close button, avoiding redundancy with the back button in a stacked navigation.
     public let shouldShowCloseButton: Bool
 
-    /// The default configuration for `CustomerCenterNavigationOptions`.
+    /// A custom handler to execute when closing the Customer Center from the close button in the navigation bar.
     ///
-    /// - `usesNavigationStack`: `true` (default to modern navigation).
-    /// - `usesExistingNavigation`: `false` (wraps content in a new navigation stack by default).
-    /// - `shouldShowCloseButton`: `true` (displays a close button by default).
-    public static let `default` = CustomerCenterNavigationOptions()
+    /// `onCloseHandler` allows developers to define a custom method for handling the dismissal of
+    /// the Customer Center. This is useful in cases where the default SwiftUI dismissal (`@Environment(\.dismiss)`)
+    /// is insufficient, such as when integrating with hybrid frameworks.
```

Comment:
this is a public API, not sure we want to mention this, it could be a bit confusing

Reply:
Do you mean  ... ?

> This is useful in cases where the default SwiftUI dismissal (`@Environment(\.dismiss)`) is insufficient, such as when integrating with hybrid frameworks.

I thought it was useful, in case someone wants to build their own thing. But if it's not a common thing then I can remove it.